From f84e858d81758167a61e28bfa63ddf1ffc8699e9 Mon Sep 17 00:00:00 2001
From: "Luis R. Rodriguez" <mcgrof@kernel.org>
Date: Thu, 20 Jul 2017 13:13:40 -0700
Subject: [PATCH 2/5] firmware: define pr_fmt
Git-commit: 73da4b4b77661ba3a9a50e5fbb412c9766149911
Patch-mainline: v4.14-rc1
References: FATE#323821

For some reason we have always forgotten this. Without this
we don't get a nice prefix on our pr_debug() / pr_*() messages.

Signed-off-by: Luis R. Rodriguez <mcgrof@kernel.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Luis R. Rodriguez <mcgrof@suse.com>
---
 drivers/base/firmware_class.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/base/firmware_class.c b/drivers/base/firmware_class.c
index 9bc89f332d29..dc2c1c8f4e4c 100644
--- a/drivers/base/firmware_class.c
+++ b/drivers/base/firmware_class.c
@@ -7,6 +7,8 @@
  *
  */
 
+#define pr_fmt(fmt) KBUILD_MODNAME ": " fmt
+
 #include <linux/capability.h>
 #include <linux/device.h>
 #include <linux/module.h>
-- 
2.14.0

